src/components/Search/SearchResultsListItem.tsx

```diff
@@ -33,16 +38,16 @@ function SearchResultsListItem(props: SearchResultsListItemProps) {
           sx={{ noWrap: 'true' }}
           primaryTypographyProps={{ noWrap: true }}
           secondaryTypographyProps={{ noWrap: true }}
-          onClick={handleChildClick}
         />
       </ListItem>
     </ListItemButton>
   );
 }
 
 interface SearchResultsListItemProps {
+  checked: string[];
```

Comment:
Hmm, is there a reason to send the whole string array here? Could we only pass a boolean `checked` value for this list item instead? I feel like this could isolate the component more.

Reply:
after making the changes in the add revision patch and moving checked revisions to the store, this will be removed.